lambdas/models/bulk_upload_report_output.py

```diff
@@ -19,6 +19,7 @@ def __init__(
         self.total_suspended = set()
         self.total_deceased = set()
         self.total_restricted = set()
+        self.total_ingested = 0
 
```

Comment:
I think we would want to follow one standard here with set() and update(). Or making it dynamic in some way